Polyhedron/demo/Polyhedron/Plugins/Mesh_3_plugin/C3t3_io_plugin.cpp

```diff
@@ -58,6 +58,7 @@ Polyhedron_demo_c3t3_binary_io_plugin::load(QFileInfo fileinfo) {
 
 
         if(item->load_binary(in)) {
+          item->c3t3_changed();
           item->changed();
           return item;
```

Comment:
Note for next week during the review of Three: I think that the API of the demo is really broken if, after a call to `item->load_binary(in)`, the user must call two other functions `item->c3t3_changed()` and `item->changed()`.

Should not `load_binary` calls those functions itself?!


Cc: @CGAL/geometryfactory